internal/pke/workflow/assemble_http_proxy_settings_activity_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	"context"
+	"testing"
+
+	"emperror.dev/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAssembleHTTPProxySettings(t *testing.T) {
```

Comment:
Although in this case this works fine, it wouldn't work if the activity made calls related to activity context  (eg. `GetLogger`). In those cases you need to use the testing package of cadence:

https://cadenceworkflow.io/docs/07_goclient/13_workflow_testing

Reply:
The person who needs to fix the tests after adding changes that break them because of stuff missing from the context shall definitely do that.